kivymd/vendor/circularTimePicker/__init__.py

```diff
@@ -510,8 +507,7 @@ def _update_start_angle(self, *a):
 
 
 class CircularHourPicker(CircularNumberPicker):
-    """:class:`CircularNumberPicker` implementation for hours.
-    """
+    """:class:`CircularNumberPicker` implementation for hours."""
 
     # military = BooleanProperty(False)
 
```

Comment:
Updated by pre-commit of github actions. 8994a39cd2b1050aa6b8342b533d148cbe2e065a